{
  "data_type": "CVE",
  "data_format": "MITRE",
  "data_version": "4.0",
  "CVE_data_meta": {
    "ID": "CVE-2022-2185",
    "ASSIGNER": "cve@gitlab.com"
  },
  "problemtype": {
    "problemtype_data": [
      {
        "description": [
          {
            "lang": "en",
            "value": "CWE-78"
          }
        ]
      }
    ]
  },
  "references": {
    "reference_data": [
      {
        "url": "https://gitlab.com/gitlab-org/cves/-/blob/master/2022/CVE-2022-2185.json",
        "name": "https://gitlab.com/gitlab-org/cves/-/blob/master/2022/CVE-2022-2185.json",
        "refsource": "CONFIRM",
        "tags": [
          "Vendor Advisory"
        ]
      },
      {
        "url": "https://hackerone.com/reports/1609965",
        "name": "https://hackerone.com/reports/1609965",
        "refsource": "MISC",
        "tags": [
          "Permissions Required",
          "Third Party Advisory"
        ]
      },
      {
        "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/366088",
        "name": "https://gitlab.com/gitlab-org/gitlab/-/issues/366088",
        "refsource": "MISC",
        "tags": [
          "Broken Link"
        ]
      }
    ]
  },
  "description": {
    "description_data": [
      {
        "lang": "en",
        "value": "A critical issue has been discovered in GitLab affecting all versions starting from 14.0 prior to 14.10.5, 15.0 prior to 15.0.4, and 15.1 prior to 15.1.1 where an authenticated user authorized to import projects could import a maliciously crafted project leading to remote code execution."
      }
    ]
  }
}